[develop]: Updated ConfigWorkflow.rst to reflect changes to config_defaults.yaml (PI13) (#1133)
ufs-community/ufs-srweather-app: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

``RUN_CMD_NEXUS``: (Default: "")
  The run command for the AQM NEXUS tasks.

> ``RUN_CMD_NEXUS``: (Default: "")
>   The run command for the AQM NEXUS tasks.
> 

Reviewer: ```suggestion
```

   System directory containing the graphics shapefiles. On Level 1 systems, these are set within the machine files. Users on other systems will need to provide the path to the directory that contains the *Natural Earth* shapefiles.

``FIXaqm``: (Default: "")
<<<<<<< HEAD:doc/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst
   System directory where AQM data files are located.

``FIXemis``: (Default: "")
   System directory where AQM emission data files are located.
=======
   Path to system directory containing AQM fixed files.

``FIXemis``: (Default: "")
   Path to system directory containing AQM emission data files.

> >>>>>>> origin/develop:doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

Reviewer: This is a merge conflict that needs to be resolved.

If non-default parameters are selected for the variables in this section, they should be added to the ``workflow:`` section of the ``config.yaml`` file. 

<<<<<<< HEAD:doc/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst
``WORKFLOW_ID``: (Default: "")
=======
``WORKFLOW_ID``: (Default: ``''``)

> >>>>>>> origin/develop:doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

Reviewer: Another merge conflict that needs to be resolved.

   Name of a file that contains settings and configurations for the :term:`NUOPC`/:term:`ESMF` main component. In general, users should not set this variable in their configuration file (see :ref:`note <tmpl-fn-warning>`).

``UFS_CONFIG_FN``: (Default: "ufs.configure")

>   Name of a template file that contains information about the various UFS components and their run sequence (ufs-weather-model: ufs.configure). Its default value is the name of the file that the ufs weather model expects to read in.

Reviewer: ```suggestion
   Name of a template file that contains information about the various :term:`UFS` components and their run sequence. In general, users should not set this variable in their configuration file (see :ref:`note <tmpl-fn-warning>`).
```
@jdkublnick I'd like to revert this definition to what is in `develop` already, tho you can keep the addition of the word "template." The default value is already listed, and the extra info in the `develop` RST file is important.

``UFS_CONFIG_FN``: (Default: "ufs.configure")
   Name of a file that contains information about the various :term:`UFS` components and their run sequence. In general, users should not set this variable in their configuration file (see :ref:`note <tmpl-fn-warning>`).

> ``UFS_CONFIG_FN``: (Default: "ufs.configure")
>   Name of a template file that contains information about the various UFS components and their run sequence (ufs-weather-model: ufs.configure). Its default value is the name of the file that the ufs weather model expects to read in.
> 

Reviewer: ```suggestion
```
This variable is listed above.

``UFS_CONFIG_TMPL_FP``: (Default: ``'{{ [user.PARMdir, UFS_CONFIG_FN]|path_join }}'``) 
   Path to the ``UFS_CONFIG_FN`` file. 

> ``UFS_CONFIG_TMPL_FP:``: (Default: ``'{{ [user.PARMdir, UFS_CONFIG_FN]|path_join }}'``)
>    Path to the ``UFS_CONFIG_FN`` file.
> 

Reviewer: ```suggestion
```
This variable is listed above.

``UFS_CONFIG_FP``: (Default: ``'{{ [EXPTDIR, UFS_CONFIG_FN]|path_join }}'``)
   Path to the ``UFS_CONFIG_FN`` file in the experiment directory. 

> ``UFS_CONFIG_FP``: (Default: ``'{{ [EXPTDIR, UFS_CONFIG_FN]|path_join }}'``)
>    Path to the ``UFS_CONFIG_FN`` file in the experiment directory. 
> 

Reviewer: ```suggestion
```
Variable already included

``GLOBAL_VAR_DEFNS_FN``: (Default: "var_defns.yaml")
<<<<<<< HEAD:doc/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst
   Name of the experiment configuration file. It contains the primary experiment variables defined in this default configuration script and in the user-specified configuration as well as secondary experiment variables generated by the experiment generation script. This file is the primary source of information used in the scripts at run time. The primary variables are defined in the default configuration file (``config_defaults.yaml``) and in the user configuration file (``config.yaml``). The secondary experiment variables are generated by the experiment generation script. 
=======
   Name of the auto-generated experiment configuration file. It contains the primary experiment variables defined in this default configuration script and in the user-specified configuration as well as secondary experiment variables generated by the experiment generation script from machine files and other settings. This file is the primary source of information used in the scripts at run time.

> >>>>>>> origin/develop:doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

Reviewer: Merge conflict that needs to be resolved.

``CCPP_PHYS_SUITE_FP``: (Default: ``'{{ [workflow.EXPTDIR, CCPP_PHYS_SUITE_FN]|path_join }}'``)
   The full path to the suite definition file (SDF) in the experiment directory. 

<<<<<<< HEAD:doc/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst
``CCPP_PHYS_DIR``: (Default: ``'{{ [user.UFS_WTHR_MDL_DIR, "FV3", "ccpp", "physics", "physics", "SFC_models", "Land", "Noahmp"] |path_join }}'``)
=======
``CCPP_PHYS_DIR``: (Default: ``'{{ [user.UFS_WTHR_MDL_DIR, "FV3", "ccpp", "physics", "physics", "SFC_Models", "Land", "Noahmp"] |path_join }}'``)

> >>>>>>> origin/develop:doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

Reviewer: Merge conflict that needs to be resolved.

> ``OMP_NUM_THREADS_RUN_FCST``: (Default: 1)

Reviewer 2: The defaults for OMP_NUM_THREADS_RUN_FCST should be 2.

```suggestion
``OMP_NUM_THREADS_RUN_FCST``: (Default: 2)
```

      For more information, see the `Intel Development Reference Guide <https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-10/thread-affinity-interface.html>`__. 

<<<<<<< HEAD:doc/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst
``OMP_NUM_THREADS_RUN_FCST``: (Default: 2)
   The number of OpenMP threads to use for parallel regions. Corresponds to the ``ATM_omp_num_threads`` value in ``nems.configure``.
=======
``OMP_NUM_THREADS_RUN_FCST``: (Default: 1)
   The number of OpenMP threads to use for parallel regions. Corresponds to the ``atmos_nthreads`` value in ``model_configure``.

> >>>>>>> origin/develop:doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

Reviewer: Merge conflict that needs to be resolved.
